index.html

```diff
@@ -738,18 +740,39 @@ <h3>Introduction</h3>
 					SVG images are not supported in the [=spine=] but can be embedded in XHTML documents.</p>
 			</section>
 
-			<section id="html-req">
-				<h3>Requirements</h3>
+			<section id="xhtml">
+				<h3>XHTML</h3>
 
-				<p>An [=eBraille content document=]:</p>
+				<p>An [=eBraille content document=] MUST be an <a data-cite="epub-33#sec-xhtml">XHTML content
+						document</a> as defined [[epub-33]] with additional constraints as defined in the following
+					subsections.</p>
 
-				<ul>
-					<li>MUST be an <a data-cite="epub-33#sec-xhtml">XHTML content document</a> [[epub-33]].</li>
-					<li>MUST NOT contain features specified in <a href="#html-no-support"></a>.</li>
-				</ul>
+				<section id="xhtml-char-encoding">
+					<h4>Character encoding</h4>
 
-				<section id="html-no-support">
-					<h3>Unsupported features</h3>
+					<p>[=eBraille content documents=] are required to provide their text content in 6- or 8-dot braille
+						characters. eBraille is not meant as a delivery format for ASCII braille or similar braille
+						encodings that require a translation to 6- or 8-dot braille characters.</p>
+
+					<p>For this reason, the text content of [=eBraille content documents=] SHOULD consist only of
+						characters in the <a href="https://www.unicode.org/charts/nameslist/c_2800.html">Braille
+							Patterns</a> [[unicode]] as well as [=ASCII whitespace=] [[infra]].</p>
```

Comment:
I don't think ASCII white space will suffice. There is the [requirement of the soft hyphen](https://github.com/daisy/ebraille/issues/51). There might also be a need for zero-width space (to create soft wrap opportunities), and no-break space (although we may also use U+2800 for that).

Reply:
The wbr tag is probably better, because more explicit.